libdokan/fs.go

```diff
@@ -274,27 +274,28 @@ func (f *FS) CreateFile(ctx context.Context, fi *dokan.FileInfo, cd *dokan.Creat
 		f.log.Errorf("FS CreateFile - Refusing real access: SID match error")
 		return openFakeRoot(ctx, f, fi)
 	}
-	f.logEnter(ctx, "FS CreateFile")
 	return f.openRaw(ctx, fi, cd)
 }
 
 // openRaw is a wrapper between CreateFile/CreateDirectory/OpenDirectory and open
 func (f *FS) openRaw(ctx context.Context, fi *dokan.FileInfo, caf *dokan.CreateData) (dokan.File, bool, error) {
 	ps, err := windowsPathSplit(fi.Path())
 	if err != nil {
+		f.log.Errorf("FS openRaw - path split error: %v", err)
```

Comment:
Please use `CErrorf` so it gets tagged with the `ctx`.

Reply:
Done here and elsewhere in libdokan.